applications/nrf5340_audio/src/modules/display/log_tab.c

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "display.h"
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/display.h>
+#include <zephyr/drivers/gpio.h>
+#include <lvgl.h>
+#include <stdio.h>
+#include <string.h>
+#include <zephyr/kernel.h>
+#include <zephyr/zbus/zbus.h>
+#include <zephyr/logging/log.h>
+#include <stdint.h>
+#include <C:\ncs\v2.4.0\nrf\applications\nrf5340_audio\src\modules\hw_codec.h>
+#include <math.h>
+#include <zephyr/logging/log_output.h>
+#include "log_tab.h"
+#include <stdbool.h>
+#include <zephyr/toolchain.h>
+#include <zephyr/logging/log_backend.h>
+#include <zephyr/logging/log_ctrl.h>
+#define LOG_MAX_LENGTH 20
+LOG_MODULE_DECLARE(display_log, CONFIG_DISPLAY_LOG_LEVEL);
+
+K_MUTEX_DEFINE(nrf5340_display_mtx);
+
+lv_obj_t *log_list;
+int cnt;
+struct backend_context {
+	uint32_t cnt;
+	const char *exp_str[10];
+	uint32_t delay;
+	bool active;
+	struct k_timer timer;
+};
+static int cbprintf_callback(int c, void *ctx)
```

Comment:
Please add some doc.

Reply:
Looks like this variable has been removed now